playground/infrastructure/helper.py

```diff
@@ -341,13 +342,13 @@ def _validate(tag: dict, supported_categories: List[str]) -> bool:
   # check that context line's value is integer
   context_line = tag.get(TagFields.context_line)
   if not isinstance(context_line, int):
-      logging.error(
-          "tag's field context_line is incorrect: %s \n"
-          "context_line variable should be integer format, "
-          "but tag contains: %s",
-          tag,
-          context_line)
-      valid = False
+    logging.error(
+        "tag's field context_line is incorrect: %s \n"
+        "context_line variable should be integer format, "
```

Comment:
```suggestion
        "Context_line variable should be integer format, "
```

Reply:
I suppose, since `context_line` is a variable, it is better to start it with a lowercase letter, what do you think?

Reply:
`context_line` is the name of the field. Maybe better to use lower-case?